pylint_venv.py

```diff
@@ -94,7 +98,7 @@ def activate_venv(venv):
             raise IncompatibleVenvError(
                 f"The virtual environment {venv!r} has multiple lib/pythonX.Y directories."
             )
-        site_packages = os.path.join(lib_dir, python_dirs[0], "site-packages")
+        site_packages = os.path.join(list(python_dirs)[0], "site-packages")
```

Comment:
non-critical/very minor:
```suggestion
        site_packages = os.path.join(next(python_dirs), "site-packages")
```
might be slightly more efficient as it avoids creating a new list.